src/actions/assets/Potree.js

```diff
@@ -0,0 +1,53 @@
+import AssetActionPlugin from "../AssetActionPlugin";
+import URI from 'urijs';
+import i18n from "../../i18n";
+
+const POTREE_SUPPORTED_TYPES = [
+  'application/vnd.laszip+copc',
+];
+
+// this.component.filename.endsWith or this.asset.href.includes
+const POTREE_SUPPORTED_FILEEXTS = [
+ 'cloud.js', 'metadata.json', 'ept.json'
+ // potree v1, potree v2, EPT Entwine Point Tiles
+];
+  
+export default class Potree extends AssetActionPlugin {
+
+  get show() {
+    return this.component.isBrowserProtocol && (
+      POTREE_SUPPORTED_TYPES.includes(this.asset.type)
+      || POTREE_SUPPORTED_FILEEXTS.map(
+        f => this.asset.href.toLowerCase().includes(f)
```

Comment:
I think we should use urijs to get the path component and then do an endsWith check. That will be more accurate.

Reply:
cf https://github.com/radiantearth/stac-browser/pull/465#discussion_r1908821847

Reply:
Just switched to `URI(this.asset.href).filename.endsWith(f)` but below comment might involve a bit more complex check

Reply:
filename() is a method according to the docs, not a property. Does it work as you posted it? Then there might be some hidden functionality in urijs for property-style access ;-)

Reply:
nope i had to use the method indeed